opentech/apply/funds/models.py

```diff
@@ -275,6 +345,7 @@ def build_json_order_by(field):
 class ApplicationSubmission(AbstractFormSubmission):
     form_data = JSONField(encoder=DjangoJSONEncoder)
     round = models.ForeignKey('wagtailcore.Page', on_delete=models.CASCADE, related_name='submissions', null=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
```

Comment:
why not use `get_user_model()` as it is used in at least one other place above.

Reply:
> When you define a foreign key or many-to-many relations to the user model, you should specify the custom model using the AUTH_USER_MODEL setting. 

We should be consistent. FK should use the settings and model references should use the `get_user model`.

`get_user model` is basically a nice wrapper for:
```
from django.utils.module_loading import import_string
import_string(settings.AUTH_USER_MODEL)
```
https://docs.djangoproject.com/en/2.0/topics/auth/customizing/#django.contrib.auth.get_user_model

Reply:
👍 missed that from the docs.